ui/component/or-asset-viewer/src/index.ts

```diff
@@ -303,12 +321,19 @@ export class OrAssetViewer extends subscribe(manager)(translate(i18next)(LitElem
     }
 
     public static getPanel(name: string, asset: Asset, attributes: AssetAttribute[], viewerConfig: AssetViewerConfig, panelConfig: PanelConfig, shadowRoot: ShadowRoot | null) {
+
         const content = OrAssetViewer.getPanelContent(name, asset, attributes, viewerConfig, panelConfig, shadowRoot);
         if (!content) {
             return;
         }
 
         return html`
+            <style>
+               .panel {
+                    position: relative;
+               }
+            </style>
+            
```

Comment:
I'm going to move this style into style.ts; keeps styles in a central place and avoids creating a style DOM node for every panel